### PR TITLE
Ali/cp 571 update models

### PR DESCRIPTION
**Pull request by ali60**
Merged all models from 
https://github.com/marqo-ai/marqo/blob/bf47a80b423d09f82002fcb6cf03b2649d849023/src/marqo/s2_inference/model_registry.py#L1478

### CODE REVIEW

File: src/marqo_commons/model_registry/model_properties_data/no_model.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    tokens: int
+    notes: str = ""
+
+
+@convert_model_properties_to_dict
+def _get_no_model_properties() -> Dict:
+    NO_MODEL_PROPERTIES = {
+        'no_model': NoModelProperties(
+            name='no_model',
+            dimensions=0,
```
Comment (wanliAlex) [inline src/marqo_commons/model_registry/model_properties_data/no_model.py:27]: Will this 0 dimension bypass the model check? E.g., requires the user to provide a dimension

**Review by wanliAlex — CHANGES_REQUESTED** [review]

File: src/marqo_commons/model_registry/model_properties_data/no_model.py (hunk 1 above)
Comment (ali60) [reply src/marqo_commons/model_registry/model_properties_data/no_model.py:27]: actually I got  this from here
https://github.com/marqo-ai/marqo/blob/bf47a80b423d09f82002fcb6cf03b2649d849023/src/marqo/s2_inference/model_registry.py#L1966-L1972

if `no_model` doesn't make sense I can remove it , regarding the dimension I'm not sure what marqo will do  if dimension is zero

Comment (ali60) [reply src/marqo_commons/model_registry/model_properties_data/no_model.py:27]: ok I can make `dimensions` as optional and not set this field

Comment (wanliAlex) [reply src/marqo_commons/model_registry/model_properties_data/no_model.py:27]: Yeah just checked and setting it to 0 will error out. Thanks for the update!

Comment (wanliAlex) [reply src/marqo_commons/model_registry/model_properties_data/no_model.py:27]: Actually, would it be better if we set it to a required field but only make it optional for this model?

Comment (wanliAlex) [reply src/marqo_commons/model_registry/model_properties_data/no_model.py:27]: All the models have a fix dimension in the model properties except this special one.

**Review by wanliAlex — CHANGES_REQUESTED** [review]

File: src/marqo_commons/model_registry/model_properties_data/no_model.py (hunk 1 above)
Comment (wanliAlex) [reply src/marqo_commons/model_registry/model_properties_data/no_model.py:27]: Nevermind, I think your solution is fine.

**Review by wanliAlex — APPROVED** [review]

**Pull request merged**